flutter-for-android.md

```diff
@@ -0,0 +1,1864 @@
+exploited to develop rich layouts that can change overtime with the same 
+children.
+
+<!-- skip -->
+{% prettify dart %}
+new Row(
+  mainAxisAlignment: MainAxisAlignment.center,
+  children: <Widget>[
+    new Text(
+      'Row One',
+    ),
+    new Text(
+      'Row Two',
+    ),
+    new Text(
+      'Row Three',
+    ),
+    new Text(
+      'Row Four',
+    ),
+  ],
+)
+{% endprettify %}
+
+<!-- skip -->
+{% prettify dart %}
+new Column(
+  mainAxisAlignment: MainAxisAlignment.center,
+  children: <Widget>[
+    new Text(
+      'Column One',
+    ),
+    new Text(
+      'Column Two',
+    ),
+    new Text(
+      'Column Three',
+    ),
+    new Text(
+      'Column Four',
+    ),
+  ],
+)
+{% endprettify %}
+
+## What is the equivalent of a RelativeLayout
+
+A RelativeLayout is used to lay your widgets out relative to each other. In 
+Flutter there are a few ways to achieve the same result.
+
+You can achieve the result of a RelativeLayout by using a combination of 
+Column, Row, and Stack widgets. You can specify rules for the widgets 
+constructors on how the children are laid out relative to the parent.
+
+[Simple example]
+
+You can also use the more complex but powerful CustomMultiChildLayout to 
+achieve a RelativeLayout.
+
+[complex code]
+
+## What is the equivalent of a ScrollView
+
+In Android a ScrollView lets you lay your widgets such that if the users' device has a 
+smaller screen than your content, they can scroll.
+
+In Flutter the easiest way to do this is using the ListView widget. This might 
+seem like overkill coming from Android, but in Flutter a ListView widget is 
+both a ScrollView and an Android ListView.
+
+<!-- skip -->
+{% prettify dart %}
+new ListView(
+  children: <Widget>[
+    new Text(
+      'Row One',
+    ),
+    new Text(
+      'Row Two',
+    ),
+    new Text(
+      'Row Three',
+    ),
+    new Text(
+      'Row Four',
+    ),
+  ],
+)
+{% endprettify %}
+
+# Gesture Detection and Touch Event Handling
+
+## How do I add an onClick listener to a widget in Flutter
+
+In Android, you can attach onClick to views such as button by calling
+the method 'setOnClickListener'.
+
+In Flutter There are two ways of adding touch listeners
+
+1. If the Widget has support for event detection you can just pass in a 
+  function to it and handle it. For example, the RaisedButton has an onPressed 
+  parameter
+
+   <!-- skip -->
+  {% prettify dart %}
+   @override
+   Widget build(BuildContext context) {
+     return new RaisedButton(onPressed: () {
+       print("click");
+     }, child: new Text("Button"));
+   }
+   {% endprettify %}
+
+2. If the Widget does not have support for event detection, you can wrap up the 
+  widget in a GestureDetector and pass in a function to the onTap parameter.
+
+   <!-- skip -->
+  {% prettify dart %}
+     @override
+     Widget build(BuildContext context) {
+       return new Scaffold(
+           body: new Center(
```

Comment:
indenting is weird here

Reply:
This one still seems off to me, even though I'm running the formatter. Let me know if it looks better though